azure_functions_worker/functions.py

```diff
@@ -78,16 +79,16 @@ def add_function(self, function_id: str,
                         func_name,
                         f'"$return" binding must have direction set to "out"')
 
+                has_explicit_return = True
                 return_binding_name = desc.type
                 assert return_binding_name is not None
 
-                has_return = True
             elif bindings.has_implicit_output(desc.type):
                 # If the binding specify implicit output binding
                 # (e.g. orchestrationTrigger, activityTrigger)
                 # we should enable output even if $return is not specified
-                has_return = True
-                return_binding_name = f'{desc.type}_ret'
+                has_implicit_return = True
+                return_binding_name = desc.type
```

Comment:
This is the line we refer to our trigger bindings (a.k.a. [python-library activity-trigger](https://github.com/Azure/azure-functions-python-library/pull/53/files#diff-6a882075a31928bc4acabddf07ba047aR56)) for serializing implicit output values.